docker/config/devnet/dev-conf.json

```diff
@@ -0,0 +1,40 @@
+{
+    "distribution": {
+        "0x0000000000000000000000000000000000000000": "8000000000000000000",
+        "0x2dfde2178daa679508828242119dcf2114038ea8": "500000000000000000",
+        "0x45c1851c2f0dc6138935857b9e23b173185fea15": "500000000000000000"
+    },
+    "unbonding_period": 15,
+    "required_council_node_stake": "1",
+    "jailing_config": {
+        "jail_duration": 86400,
+        "block_signing_window": 100,
+        "missed_block_threshold": 50
+    },
+    "slashing_config": {
+        "liveness_slash_percent": "0.1",
+        "byzantine_slash_percent": "0.2",
+        "slash_wait_period": 10800
+    },
```

Comment:
perhaps adjust `unbonding_period`, `jail_duration`, `slash_wait_period` to make it more future-proof, as it'll be invalid after https://github.com/crypto-com/chain/pull/1008 is merged (unbonding period >= jail_duration > slash_wait_period)